statsmodels/distributions/discrete.py

```diff
@@ -0,0 +1,22 @@
+import numpy as np
+from scipy.stats import rv_discrete
+from scipy.special import gammaln
+
+class genpoisson_p_gen(rv_discrete):
+    '''A generalized Poisson discrete random variable
+    '''
+    def  _rvs(self, mu, alpha, p):
```

Comment:
no, don't add the method with pass, that will overwrite the generic method and disable it.
You can try whether it works in a smoke test. I don't remember about scipy. rv_discrete detailss